monero-sys/src/lib.rs

```diff
@@ -1571,15 +1609,15 @@ impl FfiWallet {
     }
 
     /// Initialize the wallet and download initial values from the remote node.
-    /// Does not actuallyt sync the wallet, use any of the refresh methods to do that.
-    fn init(&mut self, daemon: &Daemon) -> anyhow::Result<()> {
+    /// Does not actually sync the wallet, use any of the refresh methods to do that.
+    fn init(&mut self, daemon: &Daemon, proxy_address: &str) -> anyhow::Result<()> {
```

Comment:
this should be an `Option<_>` (sorry for all the duplicates)

Reply:
damn near had a heart attack when i saw the emails